From 308aed7fc91f40db7b7f69095c1a7b777bfeb934 Mon Sep 17 00:00:00 2001
From: awiouy <awiouy@gmail.com>
Date: Thu, 6 Jul 2017 18:50:57 +0200
Subject: [PATCH] htop: remove set_escdelay - fixed in netbsd-curses 0.2.1

drop this patch at netbsd-curses bump to 0.2.1 or higher

---
 InfoScreen.c    | 1 -
 ScreenManager.c | 1 -
 2 files changed, 2 deletions(-)

diff --git a/InfoScreen.c b/InfoScreen.c
index e27494c7..28017c63 100644
--- a/InfoScreen.c
+++ b/InfoScreen.c
@@ -117,7 +117,6 @@ void InfoScreen_run(InfoScreen* this) {
 
       if (this->inc->active)
          move(LINES-1, CRT_cursorX);
-      set_escdelay(25);
       int ch = getch();
       
       if (ch == ERR) {
diff --git a/ScreenManager.c b/ScreenManager.c
index 05e1c024..5d1d07d1 100644
--- a/ScreenManager.c
+++ b/ScreenManager.c
@@ -189,7 +189,6 @@ void ScreenManager_run(ScreenManager* this, Panel** lastFocus, int* lastKey) {
       }
 
       int prevCh = ch;
-      set_escdelay(25);
       ch = getch();
 
       HandlerResult result = IGNORED;
